src-ghc/Pact/Main.hs

```diff
@@ -14,32 +14,35 @@
 -- Maintainer  :  Stuart Popejoy <stuart@kadena.io>
 --
 
-module Pact.Main
-    (
-     main
-    ) where
+module Pact.Main ( main ) where
+
+import Prelude hiding (print,putStrLn)
+
+import System.IO
+import System.Exit hiding (die)
+import System.Directory
+import System.FilePath
+#ifndef mingw32_HOST_OS
+import System.Posix.Terminal (queryTerminal)
+import System.Posix.IO (stdInput)
+#endif
+
+import qualified Options.Applicative as O
```

Comment:
Throughout these files, whenever I touched a set of imports that needed to change due to the removal of `Data.Text` from `Pact.Types.Lang`, i took the liberty of roughly sorting them. A few of the older modules were incredibly hard to read. Hope you don't mind!

Reply:
Does it do the right thing with CPP though? Looks like it but those are files I lose sleep about, otherwise I've been doing same as I go

Reply:
All the tests pass aside from the changes we need from the Type Checker. I haven't changed any of the program, just sorted out imports. That wouldn't break CPP would it?

Reply:
Trying this out. Sorting
```haskell
import Foo.Bar
#ifdef FOO
import Quux.Stuff
#else
import Bar.Baz
#endif
import Abc.Defg
```
Haha the output of `haskell-sort-imports` on this is to ... do nothing